diy_asylum_frontend/src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import { connect } from "react-redux";
 import MainView from "./components/MainView";
+import LandingPage from "./components/LandingPage";
```

Comment:
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";

Related to comment below in same file